### PR TITLE
add benchmark test data of  Intel(R) Core(TM) i7-10710U CPU

### PR DESCRIPTION
**Pull request by ZhMi**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=4106) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=4106) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=4106) it.</sub>

**Comment by ysh329** [comment]
能把Vulkan的性能也加上？

File: benchmark/README.md [hunk 1]
```diff
@@ -4322,3 +4322,176 @@ cooling_down = 0
       yolo-fastestv2  min =    3.33  max =    3.73  avg =    3.44
   vision_transformer  min =  214.91  max =  229.91  avg =  220.82
 ```
+### Intel(R) Core(TM) i7-10710U CPU @ 1.10GHz   1.61 GHz
```
Comment (ysh329) [inline benchmark/README.md:4325]: 这里是频率范围么？

File: benchmark/README.md [hunk 2]
```diff
@@ -4322,3 +4322,176 @@ cooling_down = 0
...
+test in wsl2 with ubuntu 20.04
+
+```
+../build-vulkan/benchmark/benchncnn 10 1 0 -1 0
+loop_count = 10
+num_threads = 1
+powersave = 0
+gpu_device = -1
```
Comment (ysh329) [inline benchmark/README.md:4333]: Hi, 这里的gpu_device = -1，-1在文档里是不开启，虽然你编译了，但是没有在bench的时候开启（命令行参数），具体见benchmark/readme.md

File: benchmark/README.md (hunk 1 above)
Comment (ZhMi) [reply benchmark/README.md:4325]: 处理器基本频率 1.10 GHz 可配置的 TDP-up 频率 1.60 GHz

Comment (ysh329) [reply benchmark/README.md:4325]: image
测试的时候，这里设置为最佳性能

**Pull request closed without merging**